### DISCUSSION TITLE
Option to change the say Prefix when used in console

### DISCUSSION THREAD

**Discussion by EuanM199** [PurpurMC/Purpur#841] [__Dismissed__] [answered]
Quite a niche change. However, it would be nice to change the prefix/formatting of say in the console to make it a little bit nicer instead of the normal white text.
[SERVER] message

**Comment by BillyGalbreath** [comment, marked as answer]
this is controlled on the client side using translatable components. specifically the `chat.type.announcement` component which looks like this in the language file:

```json
 "chat.type.announcement": "[%s] %s",
```

This can be edited client side using a resource pack.

**Reply by Amgelo563** [reply]
Would it be possible to change the `SERVER` name to another one though or is that stored in the client as well?
